tests/test-war-smoke/pom.xml

```diff
@@ -152,18 +93,22 @@
     </profile>
     <profile>
       <id>test.remote</id>
-      <activation>
-        <property>
-          <name>test.mode</name>
-          <value>remote</value>
-        </property>
-      </activation>
+      <properties>
+        <!-- change the build timestamp to something palatable for gcloud -->
+        <maven.build.timestamp.format>yyyyMMdd-HHmm</maven.build.timestamp.format>
+        <gcloud-projectId-file>${project.build.directory}/gcloud-projectid.properties
+        </gcloud-projectId-file>
+        <!--
+           ~ remote tests require image in gcr.io
+           ~ if local tests are enabled with remote, this image is used locally as well
+           -->
+        <jetty.test.image>gcr.io/${app.deploy.project}/jetty:${docker.tag.short}</jetty.test.image>

```

Comment:
There was an issue here because the property for docker.tag.long was based upon the value of the maven.build.timestamp variable at the time of interpolation...and that changes between the build in jetty9 and the test cases because of the change in maven.build.timestamp.format.